src/factories/nodeFlowRun.ts

```diff
@@ -59,8 +59,8 @@ export async function flowRunContainerFactory(node: RunGraphNode) {
     ])
 
     nodesContainer.visible = true
+    nodesContainer.once('rendered', () => cull())
```

Comment:
Noticed this wasn't working before. Need to wait until AFTER everything is rendered before calling `cull`

This made me realize this is highly inefficient to render things and then cull them. But its working for now. It would be much more efficient to not render things initially and then render them if they are visible. Think we could accomplish that by setting `renderable = false` on nodes when creating them (before adding them as children). Might mess with that later but for now this works well. Just a bad fps drop when you open a sub node with a lot of nodes because they must all be rendered and then once they are culled the fps jumps back up.

Reply:
Note that when a flow is first drawn, all nodes are made visible. I guess it does matter for labels in this case though.

Reply:
Right, its rendered and then culled. Which is my point here. Unless you're getting at something else?

Reply:
Na at first I thought you may be thinking about the nodes, which will all be visible at first anyway. Ignore me, haha.